html-forms/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="">
+      <div>
+        <label for="email">Email</label><br />
+        <input type="email" /><br />
+      </div>
+
```

Comment:
Instead of adding `br` elements here, I would just wrap the `input` in another `div` for the separation. It improves readability of the code and the main purpose of a `div` is to be a wrapper and since `div` is a block level element, it will start a new line on the page.
```
<div>
  <label for="email">Email</label>
  <div>
      <input type="email" />
  </div>
</div>
```